sifter/0.4.1/pom.xml

```diff
@@ -35,7 +35,7 @@
 
   <licenses>
     <license>
-      <name>Apache License, Version 2.0</name>
```

Comment:
Don't understand some of these license name changes. For example replacing the official license name `Apache License, Version 2.0` with `Apache-2.0`?